aten/src/ATen/native/quantized/affine_quantizer.cpp

```diff
@@ -314,6 +386,27 @@ CAFFE2_API float dequantize_val(double scale, int64_t zero_point, T value) {
 }
 #endif // USE_FBGEMM
 
+/*
+* Quantize value based on the following equation
+* Xq = Round(Xf * inv_scale + zero_point)
+* where zero_point is in float.
+*
+* Note: For the case of embedding quantization we will set zero_point
+* to (-Xmin/scale), where Xmin is the min value in input tensor row.
+*/
+template <typename T>
+T quantize_val_float_qparams(float scale, float zero_point, float value) {
```

Comment:
@vkuzo, @raghuramank100 I've updated the float qparams function to use the same equation as our current quantize function. Please take a look 
The zero_point passed in will be (-Xmin/scale), so that we can match the caffe2 quantize function used here - https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/cpu/qembeddingbag_prepack.cpp#L119-L122